src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -15,8 +19,8 @@ public User login(String login, String password) throws AuthenticationException
     }
 
     private User findByLogin(String login) {
+        logger.info("User with login: " + login + " successfully found");
```

Comment:
If you write that user with login was successfully found, then, maybe, it is better to log it before return statement, when all method logic is complete?